### PR TITLE
Feature transform matrix to raw echelon form

### PR DESCRIPTION
**Pull request by lioshik**
In this extension, I lacked the function of bringing the matrix to an raw echelon form, so I decided to add it. To make this transformation, you need to select the matrix and then press  Shift + Ctrl + Alt + T  (or Shift + Cmd + Alt + T on Mac). I also added description of this functionality in README.md with examples

### CODE REVIEW

**Comment by Mality** [comment]
It's a very useful feature!

File: server.py [hunk 1]
```diff
@@ -28,6 +28,20 @@ def get_latex():
             'error': str(e)
         }
 
+@app.route('/matrix-raw-echelon-form', methods=['POST'])
+def get_matrix_raw_echelon_form():
+    try:
+        latex(latex2sympy(request.json['data']).subs(variances).rref()[0])
```
Comment (Mality) [inline server.py]: Why this line is duplicated here?

Comment (lioshik) [reply server.py]: Yes, this line here doesn't make sense. I removed it

**Comment by OrangeX4** [comment]
Thanks for your PR!

I will check it and update it soon.

**Pull request merged**